packages/design-system-docs/src/pages/components/Alert/_Alert.docs.scss

```diff
@@ -1,6 +1,8 @@
 /*
 Alert
 
+@status Ready
```

Comment:
Is there a particular reason why the Ready status is being added to each component? I know that each one is not in Draft, but I was under the impression it would be good to use for newly Ready components, and not necessarily every single component

Reply:
Yeah good point I'm removing them